CONTRIBUTING.md

```diff
@@ -37,6 +37,19 @@ License Agreement before we can accept your pull request: https://cla.puppet.com
 1. Add the puppetlabs repo as an upstream - `git remote add upstream git@github.com:puppetlabs/bolt`
 1. Make a new branch off of master - `git checkout -b mybranchname`
 1. Commit your changes and add a useful commit message, including what specifically you changed and why - `git commit`
+    * If your changes are user-facing, add a release note to the end of a commit message. Release notes should begin
+      with a label indicating what kind of change you are making. Valid labels include `!feature`, `!bug`, `!deprecation`,
+      and `!removal`. 
+      
+      Release notes should follow this format:
+
+      ```
+      !label
```

Comment:
I'd feel better about this if we had a workflow that checked for a release note, and failed if none was provided. `!no-release-note` would be valid for commits that don't require release.

Reply:
I think that gets tricky when you have a PR with more than one commit, as it would require that you add a label to _every_ commit message.